andela/wg-silicon
# [Chore #158626103]relate workout log and session

#### What does this PR do?
Creates a relationship between workout log and workout session using a workoutsession_id foreign key on workoutlog table from workoutsession table

#### Description of Task to be completed?
Right now the workout logs and the workout session are only linked through the date (for sessions this value is unique and enforced by the DB). This presents in some places a problem, since it's not possible to just follow a foreign key relationship with the ORM like in other places. 
To fix this either:

a) see if it's possible to define the date as the foreign key in django
or
b) check if adding a real foreign key from the logs to the session makes sense

#### How should this be manually tested?
The update only affects the database. Ensure to run migrations to enable the changes take effect on the database. 
``` ./manage.py makemigrations ```

Run the tests using the command:
``` ./manage.py test```
or
#### What are the relevant pivotal tracker stories?
[#158626103](https://www.pivotaltracker.com/story/show/158626103)
